utils/benchmark-operator.sh

```diff
@@ -65,9 +65,12 @@ run_benchmark() {
       kubectl logs -n benchmark-operator --prefix --tail=30 ${pod}
       kubectl logs -n benchmark-operator --prefix --tail=-1 ${pod} >> ${pod_log}
     done
-    remove_cli
   fi
   local benchmark_name=$(cat ${1} | python -c 'import yaml; import sys; print(yaml.safe_load(sys.stdin.read())["metadata"]["name"])')
   gen_metadata ${benchmark_name} ${start_date} $(date +%s%3N)
+
+  if [[ -${rc} -eq 1 ]]; then
```

Comment:
Why did you add this conditional?

Reply:
because the remove_cli was originally only in the if statement starting on line 57, so I was trying to keep the same functionality by putting an if statement

Reply:
@rsevilla87 do you think we could move the _remove_cli_ to the end of the run.sh scripts in case other places need items installed by the benchmark operator requirements? There might also be other cases similar to the one I hit if we remove the cli but need a package that was installed in the virtualenv

Do we need to be calling remove_benchmark_operator instead?

Reply:
Specifically for the concurrent-builds it runs _run_benchmark_ multiple times so I would have to add in a call to re-activate the cli